client/analytics/ad-tracking.js

```diff
@@ -62,15 +59,36 @@ const FACEBOOK_TRACKING_SCRIPT_URL = 'https://connect.facebook.net/en_US/fbds.js
 /**
  * Globals
  */
-if ( ! window._fbq ) {
-	window._fbq = []; // Facebook global
+if ( ! window.fbq ) {
+	setUpFacebookGlobal();
 }
 
 if ( ! window.uetq ) {
 	window.uetq = []; // Bing global
 }
 
+function setUpFacebookGlobal() {
+	const facebookEvents = window.fbq = function() {
+		if ( facebookEvents.callMethod ) {
+			facebookEvents.callMethod.apply( facebookEvents, arguments );
+		} else {
+			facebookEvents.queue.push( arguments );
+		}
+	};
+
+	if ( ! window._fbq ) {
+		window._fbq = facebookEvents;
+	}
+
+	facebookEvents.push = facebookEvents;
+	facebookEvents.loaded = true;
+	facebookEvents.version = '2.0';
+	facebookEvents.queue = [];
```

Comment:
This is a beautified version of the setup in the Facebook tracking pixel ([docs](https://www.facebook.com/business/help/952192354843755)). I'll add a comment for that.